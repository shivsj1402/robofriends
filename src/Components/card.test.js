import React from "react";
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Card from "./card";

it('card component', ()=>{
    expect(toJson(shallow(<Card />))).toMatchSnapshot();
})
