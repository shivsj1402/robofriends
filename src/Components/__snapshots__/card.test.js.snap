// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card component 1`] = `
<div
  className="tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5"
>
  <img
    alt="Robot_image"
    src="https://robohash.org/undefined?200*200"
  />
  <h2 />
  <p />
</div>
`;
